package.json

```diff
@@ -1,10 +1,13 @@
 {
   "name": "@magnet.me/consultant",
-  "version": "1.0.0",
+  "version": "1.0.2",
   "description": "A library to register services to Consul and listen for property changes.",
-  "main": "src/index.js",
+  "main": "out/index.js",
```

Comment:
Interesting. So far I've only seen `out` used here at magnet.me, so I'd rather keep it consistent with our own stack.